src/jit/codegenxarch.cpp

```diff
@@ -3563,7 +3579,7 @@ void CodeGen::genStructPutArgUnroll(GenTreePutArgStk* putArgNode, unsigned baseV
     noway_assert(putArgNode->TypeGet() == TYP_STRUCT);
```

Comment:
Doesn't this need to be varTypeIsStruct()?  As per my understanding this method is used for passing a struct arg on stack by unrolling copy-loop.  SIMD types args never passed by using this method or just none of the tests have hit this code path?